debian/changelog

```diff
@@ -1,6 +1,6 @@
 system76-driver (20.04.37~~alpha) focal; urgency=low
 
-  * Daily WIP for 20.04.37
+  * Adds an uptime output file for support
```

Comment:
You need to keep the Daily WIP line, and add a _different_ entry for the change.

Reply:
Ah sorry about that! Should be good now!